keras_hub/src/models/differential_binarization/differential_binarization_presets.py

```diff
@@ -0,0 +1,17 @@
+"""Differential Binarization preset configurations."""
+
+backbone_presets = {
+    "diffbin_r50vd_icdar2015": {
```

Comment:
I think we should use consistent abbreviations. If the class is `DifferentialBinarizationXX`, then the preset name would be `"differential_binarization"`. Or we could do `DiffBinBackbone` and `diff_bin_....` for the preset name.

Part of me is actually tempted by the latter, since `DifferentialBinarization` is such a mouthful. But I am not sure what is common in other libraries? Is there a common pattern in other projects?

Reply:
PaddleOCR mostly uses "DB", but I find this abbreviation somewhat indistinctive (in the sense you have a hard time googling it). I've also seen DBNet, but I don't know where this comes from, as the paper never names it DBNet, but just "Differential Binarization".

Agreed that `DifferentialBinarization` is quite long though and consistency is always good.

Reply:
Let's go with DiffBin everywhere!